src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -13,6 +13,7 @@
 import org.intellij.sdk.codesync.CodeSyncLogger;
 import org.intellij.sdk.codesync.NotificationManager;
 import org.intellij.sdk.codesync.auth.CodeSyncAuthServer;
+import org.intellij.sdk.codesync.clients.ClientUtils;
```

Comment:
What is this import doing?

Reply:
Believe we had added this import in our previous implementations, we are not using it now. Removing